File: 1ºAno/Lista1/C/Sala/files/ex1.c
#include <stdio.h>

int main() {

    float dados[4];
    printf ("Tempo gasto (em h): ");
    scanf("%f", &dados[0]);
    printf ("Velocidade média (em km/h): ");
    scanf("%f", &dados[1]);
    dados[2] = dados[0] * dados[1];
    dados[3] = dados[2] / 12;
    printf ("\nTempo gasto: %.2f h\n", dados[0]);
    printf ("Velocidade média: %.2f km/h\n", dados[1]);
    printf ("Distância: %.2f km\n", dados[2]);
    printf ("Litros usados: %.2f\n", dados[3]);

    return 0;
}

File: 1ºAno/Lista1/Python/Sala/files/ex1.py
dados = {'Tempo gasto (em h):' : float(input('Tempo gasto (em h): ')), 'Velocidade média (em km/h):': float(input('Velocidade média (em km/h): '))}
dados.update({'Distância (em km):': dados['Tempo gasto (em h):'] * dados['Velocidade média (em km/h):']})
dados.update({'Litros usados:' : dados['Distância (em km):'] / 12})
print ('')
for i in dados:
    print (i, "{:.2f}" .format(dados[i]))